pkg/graveler/committed/batch_write_closer.go

```diff
@@ -5,20 +5,21 @@ package committed
 // BatchWriterCloser collects Range writers and handles the asynchronous
 // flushing and closing of the writers.
 // Example usage:
-// func batch(manager RangeManager, bwc BatchWriterCloser) {
-//	w1, _ := manager.GetWriter()
-//	_ = w1.WriteRecord(graveler.ValueRecord{Key: "foo1", Value: &graveler.Value{Address: "bar1"}})
-//	_ = w1.WriteRecord(graveler.ValueRecord{Key: "foo2", Value: &graveler.Value{Address: "bar2"}})
-//	_ = bwc.CloseWriterAsync(w1)
-
-//	w2, _ := manager.GetWriter()
-//	_ = w2.WriteRecord(graveler.ValueRecord{Key: "goo1", Value: &graveler.Value{Address: "baz1"}})
-//	_ = bwc.CloseWriterAsync(w2)
-
-//	// blocks until all writers finished or any writer failed
-//	res, err := bwc.Wait()
-//	// handle err, results, etc..
-// }
+//
+//	func batch(manager RangeManager, bwc BatchWriterCloser) {
```

Comment:
Why do we need all this whitespace?

Reply:
Part of the changes introduced in Go fmt 1.19 - https://tip.golang.org/doc/go1.19.
When you run `gofmt -s -w <file>` it will apply the above on existing code comments.